{"name": "test_login_mobile_error", "status": "broken", "statusDetails": {"message": "TypeError: test_login_mobile_error() missing 2 required positional arguments: 'user_data' and 'init_app'", "trace": "self = <unittest.case._Outcome object at 0x0000000004797DD8>\ntest_case = <test_login.TestLogin testMethod=test_login_mobile_error>\nisTest = True\n\n    @contextlib.contextmanager\n    def testPartExecutor(self, test_case, isTest=False):\n        old_success = self.success\n        self.success = True\n        try:\n>           yield\n\nC:\\Program Files\\Python37\\lib\\unittest\\case.py:59: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <test_login.TestLogin testMethod=test_login_mobile_error>\nresult = <TestCaseFunction test_login_mobile_error>\n\n    def run(self, result=None):\n        orig_result = result\n        if result is None:\n            result = self.defaultTestResult()\n            startTestRun = getattr(result, 'startTestRun', None)\n            if startTestRun is not None:\n                startTestRun()\n    \n        result.startTest(self)\n    \n        testMethod = getattr(self, self._testMethodName)\n        if (getattr(self.__class__, \"__unittest_skip__\", False) or\n            getattr(testMethod, \"__unittest_skip__\", False)):\n            # If the class or method was skipped.\n            try:\n                skip_why = (getattr(self.__class__, '__unittest_skip_why__', '')\n                            or getattr(testMethod, '__unittest_skip_why__', ''))\n                self._addSkip(result, self, skip_why)\n            finally:\n                result.stopTest(self)\n            return\n        expecting_failure_method = getattr(testMethod,\n                                           \"__unittest_expecting_failure__\", False)\n        expecting_failure_class = getattr(self,\n                                          \"__unittest_expecting_failure__\", False)\n        expecting_failure = expecting_failure_class or expecting_failure_method\n        outcome = _Outcome(result)\n        try:\n            self._outcome = outcome\n    \n            with outcome.testPartExecutor(self):\n                self.setUp()\n            if outcome.success:\n                outcome.expecting_failure = expecting_failure\n                with outcome.testPartExecutor(self, isTest=True):\n>                   testMethod()\nE                   TypeError: test_login_mobile_error() missing 2 required positional arguments: 'user_data' and 'init_app'\n\nC:\\Program Files\\Python37\\lib\\unittest\\case.py:615: TypeError"}, "start": 1569405210609, "stop": 1569405210609, "uuid": "7873ef2c-bd98-46ee-8a5e-5d2362fa2137", "historyId": "d35d63cdd582c564cf0b73c7f2c3c2c6", "testCaseId": "74d5c1ba81af3e009174da5d13236a42", "fullName": "test_cases.test_login.TestLogin#test_login_mobile_error", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PCDALAO"}, {"name": "thread", "value": "16876-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_login"}]}